app/scripts/app.js

```diff
@@ -1,170 +1,182 @@
-(function(global) {
+(function (global) {
   'use strict';
 
+  const KEY_UP = 38;
```

Comment:
`const` is ES6 and not supported in ES5. replace all `const` with `var` instead to guarantee a cross-browser support.